acos_client/v30/slb/virtual_port.py

```diff
@@ -87,7 +87,20 @@ def _set(self, virtual_server_name, name, protocol, port,
                 port_number=port, protocol=protocol
             )
 
-        return self._post(url, params, **kwargs)
+	try:
+            return self._post(url, params, **kwargs)
+        except ae.AxapiJsonFormatError as e:
+            # Workaround for 4.0.3
+            # Try again without the nulls
+            changed = False
+            for exclude_key in exclude_minimize:
+                if params["port"].get(exclude_key, '') is None:
+                    del params["port"][exclude_key]
+                    changed = True
+            if changed:
+                return self._post(url, params, **kwargs)
+            else:
+                raise e
```

Comment:
-1, this is in the wrong place.  The try belongs in update, where the resulting code is also much simpler, since the second call to _set in the except can just not pass exclude_minimize.